stdlib/public/SwiftShims/RefCount.h

```diff
@@ -1248,7 +1252,15 @@ class RefCounts {
   // Return weak reference count.
   // Note that this is not equal to the number of outstanding weak pointers.
   uint32_t getWeakCount() const;
-  
+
+  // DO NOT TOUCH.
+  // This exists for the benefits of the Refcounting.cpp tests. Do not use it
+  // elsewhere.
+  auto getBitsValue()
+    -> decltype(refCounts.load(std::memory_order_relaxed).getBitsValue()) {
```

Comment:
I believe `decltype(auto)` will get you this behavior nowadays.

Reply:
I thought that you needed C++14 for return type inference on a method.

Reply:
https://github.com/apple/swift/commit/8246864260eb6ee8e7ee62f2c60e58a9d428ff1e

Reply:
I'm behind the times! Thanks.